be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -1004,6 +1004,23 @@ Status VOlapTableSink::find_tablet(RuntimeState* state, vectorized::Block* block
         is_continue = true;
         return status;
     }
+    if (!(*partition)->is_mutable) {
+        RETURN_IF_ERROR(state->append_error_msg_to_file(
+                []() -> std::string { return ""; },
+                [&]() -> std::string {
+                    fmt::memory_buffer buf;
+                    fmt::format_to(buf, "partition immutable for this tuple. tuple={}",
```

Comment:
```suggestion
                    fmt::format_to(buf, "partition is immutable for this tuple. tuple={}",
```

Reply:
And if you call `append_error_msg_to_file`, it may return `stop_processing` to true, and stop the load job.
But in your design, these rows should be `unselected rows`, and should not stop the job

Reply:
fixed